metabase/metabase
# Entity picker improvements

Closes https://github.com/metabase/metabase/issues/50526

### Description
Makes some UI adjustments to the Entity Picker Modal, `<DatabaseList>` component, and adds a concept of tabs working with multiple modals at a time, while being able to filter what is seen. Filtering is all done on the FE.

### How to verify

1. New question -> Collections
2. Try out the new filtering button to restrict what models are returned
3. Search for something. The filter should carry over and you should only see what is returned

### Demo
![image](https://github.com/user-attachments/assets/b569b47f-df88-42f0-89be-6ace82ac01f0)

### Checklist
